frontend/src/pages/Player/RightPlayerPanel/style.css.ts

```diff
@@ -4,7 +4,6 @@ export const RIGHT_PANEL_WIDTH = 300
 
 export const playerRightPanelContainer = style({
 	position: 'relative',
-	zIndex: 5,
```

Comment:
Not needed anymore after the refactoring of the centerColumn from https://github.com/highlight/highlight/pull/4107